kibana.yml

```diff
@@ -32,6 +32,8 @@
 # is proxied through the Kibana server.
 # elasticsearch.username: "user"
 # elasticsearch.password: "pass"
+%ELASTICSEARCH_USERNAME%
```

Comment:
@jits This is our convention. `ELASTICSEARCH_USERNAME` env variable replaces a placeholder of the same name `%ELASTICSEARCH_USERNAME%`.